src/Html.php

```diff
@@ -5003,7 +5003,7 @@ public static function image($path, $options = [])
         }
 
         if (!isset($options['alt'])) {
-            $options['alt'] = $options['title'];
+            $options['alt'] = Html::resume_text($options['title'], 30);
```

Comment:
Is this possible to handle this using a CSS rule instead?

Reply:
I don't think so.